tests/helpers.bash

```diff
@@ -130,9 +130,9 @@ function teardown_tests() {
 function normalize_image_name() {
     for img in "$@"; do
         if [[ "${img##*/}" == "$img" ]] ; then
-            echo -n docker.io/library/"$img"
-        elif [[ docker.io/"${img##*/}" == "$img" ]] ; then
-            echo -n docker.io/library/"${img##*/}"
+            echo -n quay.io/libpod/"$img"
+        elif [[ quay.io/libpod/"${img##*/}" == "$img" ]] ; then
+            echo -n quay.io/libpod"${img##*/}"
```

Comment:
@edsantiago I think maybe [these test-failures](https://cirrus-ci.com/build/4628220696330240) are caused by my changes here.  Should I have done:

```suggestion
            echo -n quay.io/libpod/"${img##*/}"
```

?

Reply:
If I'm reading it right, can't this clause be dropped?

Reply:
Ed seems busy, let me just try removing it...

Reply:
Ed seems busy, let me just try removing it...

Reply:
No, I just have nothing to add. This is not my code, I don't even understand what it does or is supposed to do. I stand by anything @nalind says about it.

Reply:
Oh okay, thanks for the update Ed.